pydocx/test/utils.py

```diff
@@ -282,8 +282,8 @@ def __init__(self, *args, **kwargs):
         super(XMLDocx2Html, self).__init__(path=None, *args, **kwargs)
 
     def _load(self):
-        self.document = WordprocessingDocument(path=None)
-        package = self.document.package
+        document = WordprocessingDocument(path=None)
```

Comment:
Yes, unfortunately it is necessary. The reason we can't do that is because the document hasn't been built yet. Calling self.document will call load_document, which in turn queries for style_definitions_part which depends on main_document_part. This triggers the container to load/process the parts if it hasn't already. This document doesn't have any parts, so it becomes empty. Adding the stuff below doesn't cause the load parts to be invalidated so it isn't re-triggered. I didn't want to have to rewrite that now, so I worked around it.